azurerm/internal/services/web/tests/resource_arm_app_service_slot_test.go

```diff
@@ -1350,6 +1350,58 @@ func TestAccAzureRMAppServiceSlot_httpBlobStorageLogs(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMAppServiceSlot_detailedErrorMessagesLogs(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_app_service_slot", "test")
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMAppServiceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMAppServiceSlot_detailedErrorMessagesEnabled(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMAppServiceExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "logs.0.detailed_error_messages_enabled", "true"),
```

Comment:
Hi @jackofallops, thanks for the reviews and comments on this PR. :) I would just like to clarify this part. Do I have to remove this line `resource.TestCheckResourceAttr(data.ResourceName, "logs.0.detailed_error_messages_enabled", "true")` because the `data.ImportStep()` is already doing the checking? Sorry for this, I'm very new here. Asking for your guidance on this. Below would be resulting edited function. Thanks. :)

```
func TestAccAzureRMAppServiceSlot_detailedErrorMessagesLogs(t *testing.T) {
	data := acceptance.BuildTestData(t, "azurerm_app_service_slot", "test")
	resource.ParallelTest(t, resource.TestCase{
		PreCheck:     func() { acceptance.PreCheck(t) },
		Providers:    acceptance.SupportedProviders,
		CheckDestroy: testCheckAzureRMAppServiceDestroy,
		Steps: []resource.TestStep{
			{
				Config: testAccAzureRMAppServiceSlot_detailedErrorMessages(data, true),
				Check: resource.ComposeTestCheckFunc(
					testCheckAzureRMAppServiceExists(data.ResourceName),
				),
			},
			data.ImportStep(),
			{
				Config: testAccAzureRMAppServiceSlot_detailedErrorMessages(data, false),
				Check: resource.ComposeTestCheckFunc(
					testCheckAzureRMAppServiceExists(data.ResourceName),
				),
			},
			data.ImportStep(),
		},
	})
}
```

Reply:
Hi @daxsorbito - Yes, that's correct.  The `data.ImportStep` handles checking the attributes are set as per config.  There are cases where an explicit check may be required, typically where values are `Computed` based on other settings, and in data sources, but I believe this should be fine to remove.